src/modules/dashboard/components/DomainDropdown/CreateDomainButton.tsx

```diff
@@ -43,9 +44,15 @@ const CreateDomainButton = ({ colony }: Props) => {
   );
 
   const text = formatMessage(MSG.buttonCreateNewDomain);
-
+  const isSupportedColonyVersion =
+    colony.version >= ColonyVersion.CeruleanLightweightSpaceship;
   return (
-    <button className={styles.main} onClick={handleClick} type="button">
+    <button
+      className={styles.main}
+      onClick={handleClick}
+      type="button"
+      disabled={!isSupportedColonyVersion}
+    >
```

Comment:
The styles for this, when disabled, are awful :)

They don't tell you in any way that this is actually disabled.

Leave it like this for now, but open a issue to track fixing this, as this is not a deal-breaker :)

Reply:
https://linear.app/colony/issue/DEV-170/ui-improvements-in-disabled-state-for-colony-version-3-and-4 added to scope of this